app/pb_kits/playbook/pb_button/docs/_button_default.jsx

```diff
@@ -3,18 +3,24 @@ import {Button} from '../../'
 
 const ButtonDefault = () => (
     <div>
-        <Button text="Button Primary" />
         <Button
+                onClick={() => console.log("button clicked!")}
```

Comment:
You had mentioned something about allowing many different (or any) `on...` props for inputs. Should we consider that pattern starting here and what might it be named? Maybe `eventProps`, ect..?

Reply:
I guess it depends on how elaborated we'd like the solution to be... For instance, if prop forwarding solves the problem just fine, then the solution is somewhat trivial and I could sneak that in this PR, example.:

```jsx
const Button = ({ prop1, prop2, text, ...props }) => {
  return (
    <div className="my-complex-button">
      <button {...props}>{text}</button>
    </div>
  )
}
```

Thoughts?

Reply:
I think this might actually work just fine based on what we did back in `component/nitro-react` for supporting the Credit App.